### PR TITLE
[Docs] Default value for verification flags is 'SSL_VERIFY_NONE'

### PR DESCRIPTION
**Pull request by ruslo**
- [x] documentation is added or updated

### CODE REVIEW

**Review by paulidale — APPROVED** [review]

File: doc/man3/SSL_CTX_set_verify.pod [hunk 1]
```diff
@@ -144,6 +144,9 @@ B<Client mode:> ignored (see BUGS)
 
 If the B<mode> is SSL_VERIFY_NONE none of the other flags may be set.
 
+If the B<mode> is not modified by C<SSL_CTX_set_verify()> or
+C<SSL_set_verify()>, the behavior is similar to SSL_VERIFY_NONE.
+
```
Comment (t8m) [inline doc/man3/SSL_CTX_set_verify.pod:149]: Not just similar. I.e., `If the B<mode> is not modified ..., it defaults to B<SSL_VERIFY_NONE>`.

**Review by t8m — CHANGES_REQUESTED** [review]

File: doc/man3/SSL_CTX_set_verify.pod (hunk 1 above)
Comment (ruslo) [reply doc/man3/SSL_CTX_set_verify.pod]: Reworded, let me know if the new version sounds better

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
@paulidale please reconfirm

**Review by shahsb — APPROVED** [review]

**Comment by paulidale** [comment]
yes, good

**Comment by t8m** [comment]
Merged to all the active branches. Thank you for your contribution.

**Pull request closed without merging**